Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserTableModel.cpp

```diff
@@ -105,6 +116,13 @@ namespace AzToolsFramework
             return !parent.isValid() ? m_indexMap.size() : 0;
         }
 
+        void AssetBrowserTableModel::timerEvent(QTimerEvent* /*event*/)
```

Comment:
nit: Prefer `[[maybe_unused]]` now (more consistent and handy if people do want to check the value of `event` in a debugger)